packages/mqttsuite/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/SNodeC/mqttsuite
+TERMUX_PKG_DESCRIPTION="MQTTSuite: A lightweight MQTT Integration System"
+TERMUX_PKG_LICENSE="MIT OR LGPL-3.0-or-later"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
+TERMUX_PKG_MAINTAINER="VolkerChristian"
+TERMUX_PKG_VERSION=1.0
+TERMUX_PKG_SRCURL="git+https://github.com/SNodeC/mqttsuite.git"
```

Comment:
We *usually* prefer building from release tarballs.
```suggestion
TERMUX_PKG_SRCURL="https://github.com/SNodeC/mqttsuite/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
```